src/content/reference/react/useRef.md

```diff
@@ -532,35 +532,35 @@ function Video() {
   // ...
 ```
 
-Here, the `playerRef` itself is nullable. However, you should be able to convince your type checker that there is no case in which `getPlayer()` returns `null`. Then use `getPlayer()` in your event handlers.
+Di sini, `playerRef` itu sendiri dapat bernilai null. Namun, Anda harus dapat meyakinkan pemeriksa tipe Anda bahwa tidak ada kasus di mana `getPlayer()` mengembalikan `null`. Kemudian gunakan `getPlayer()` di dalam event handler Anda.
 
 </DeepDive>
 
 ---
 
-## Troubleshooting {/*troubleshooting*/}
+## Pemecahan masalah {/*troubleshooting*/}
 
-### I can't get a ref to a custom component {/*i-cant-get-a-ref-to-a-custom-component*/}
+### Saya tidak bisa mendapatkan ref ke komponen khusus {/*i-cant-get-a-ref-to-a-custom-component*/}
 
-If you try to pass a `ref` to your own component like this:
+Jika Anda mencoba mengoper `ref` ke komponen Anda sendiri seperti ini:
 
 ```js
 const inputRef = useRef(null);
 
 return <MyInput ref={inputRef} />;
 ```
 
-You might get an error in the console:
+Anda mungkin mendapatkan error di konsol:
 
 <ConsoleBlock level="error">
 
 Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?
 
 </ConsoleBlock>
 
-By default, your own components don't expose refs to the DOM nodes inside them.
+Secara default, komponen Anda sendiri tidak mengekspos ref ke node DOM di dalamnya.
```

Comment:
```suggestion
Secara bawaan, komponen Anda sendiri tidak mengekspos ref ke node DOM di dalamnya.
```

Reply:
@mazipan @resir014 This word needs to be confirmed in the [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium) later.

Discussion: https://github.com/reactjs/id.react.dev/discussions/337